core/src/main/java/co/aikar/commands/annotation/Subcommand.java

```diff
@@ -26,6 +26,10 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
+ * Defines the member this is attached to a subcommand of the parent.
```

Comment:
Defines the subcommand that can be used to execute this command. this is appended onto the root command for the command group, as well as any parent command groups subcommand base.